tests/util/network.js

```diff
@@ -0,0 +1,34 @@
+const JEKYLL = 'http://localhost:4000';
+const PROD = 'https://www.ada.gov';
+
+const CHECK_EXTERNAL_LINKS = process.env.CHECK_EXTERNAL_LINKS?.toLowerCase() === 'true';
```

Comment:
Initially, CHECK_EXTERNAL_LINKS will be false, right? Shouldn't process.env.CHECK_EXTERNAL_LINKS?.toLowerCase() be undefined since it hasn't been added to the user environment yet?